doc/api/cli.md

```diff
@@ -425,6 +425,22 @@ added: REPLACEME
 
 Use this flag to enable [ShadowRealm][] support.
 
+### `--experimental-synchronousworker`
```

Comment:
```suggestion
### `--experimental-synchronous-worker`
```

Reply:
Since the new class is all one `SynchronousWorker`, I'd prefer the cli flag to match, without the additional hyphen

Reply:
What's need of adding the flag? This is not a new global or a new module. It's an API on a existing module.